plugins/parsers/influx/parser.go

```diff
@@ -100,11 +100,7 @@ func (p *Parser) Parse(input []byte) ([]telegraf.Metric, error) {
 			}
 		}
 
-		metric, err := p.handler.Metric()
```

Comment:
@powersj I was trying to contrast this change with the Graphite `Parser` and the `socketlistener` (which uses the parsers). 
The [Graphite parser](https://github.com/influxdata/telegraf/blob/master/plugins/parsers/graphite/parser.go#L82) continues to return both errors and metrics. However, the [socket_listener](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/socket_listener/socket_listener.go#L53) ignores the metrics if there are errors found.

Shouldn't the socket_listener try to accept the metrics that it received and also that the influx parser should send back the errors as well?